### PR TITLE
Use exec() and print() functions in both Python 2 and Python 3

### PR DESCRIPTION
**Pull request by cclauss**
Legacy __exec__ and __print__ statements are syntax errors in Python 3 but __exec()__ and __print()__ functions works as expected in both Python 2 and Python 3.

### CODE REVIEW

File: risk_models/strategy.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ class BoolStrategy(Strategy):
 
     def __init__(self, d):
         super(BoolStrategy, self).__init__(d)
-        self.func_name = d['strategy_func']
+        self.__name__ = d['strategy_func']
```
Comment (leohowell) [inline risk_models/strategy.py]: `func_name` can not be changed here, it's not a build in attribute name but a custom attribute.

File: www/core/lru.py [hunk 2]
```diff
@@ -133,7 +133,7 @@ def clear(self):
         self.__access_times.clear()
 
     def __contains__(self, key):
-        return self.has_key(key)
```
Comment (leohowell) [inline www/core/lru.py]: `has_key` is custom method here.

**Comment by leohowell** [comment]
already Python3

**Comment by cclauss** [comment]
Nice!  /www/core/lru.py:136:20: W601 .has_key() is deprecated, use 'in'

**Comment by leohowell** [comment]
@cclauss Thanks for all your works, we appreciate it. ❤️

**Pull request closed without merging**